underscore.js

```diff
@@ -865,7 +865,7 @@
       if (callNow) {
         timeout = setTimeout(later, wait);
         result = func.apply(this, args);
-      } else if (!immediate) {
```

Comment:
This will schedule a new timeout every time the function is called when `immediate = true`.